### PR TITLE
HTML, CSS & JS - CAPSTONE PROJECT

### PR DESCRIPTION
**Pull request by chukwuemeka1234**
**In this project, I did the following:**

- [ ] Added the home page and about page
- [ ] Styled the home page and about page to be responsive to a desktop and mobile device using the mobile-first approach
- [ ] Added some basics functionality with javascript to aid interactivity for UX
- [ ] Made the webpage accessible to enhance accessibility

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    </section>
+  </main>
+
+  <footer class="footer">
+    <div class="footer-container">
+      <a class="footer-logo-container" href="#"><img class="footer-logo" src="./images/home-logo.jpg"
+          alt="muson logo"></a>
+      <small class="footer-desc">&copy; Muson 2021. Some Rights</small>
+    </div>
+  </footer>
```
Comment (rotimiazeez) [inline about.html]: - [ ] Kindly make sure that on the `mobile screen` the footer has a white background(already did👍) and on `desktop screen` it has a dark background😉 for both pages👍.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        <img src="./images/airbnb.svg" alt="airbnb">
+      </div>
+    </section>
+  </main>
+
+  <footer class="footer">
+    <div class="footer-container">
+      <a class="footer-logo-container" href="#"><img class="footer-logo" src="./images/home-logo.jpg"
+          alt="muson logo"></a>
+      <small class="footer-desc">&copy; Muson 2021. Some Rights</small>
```
Comment (rotimiazeez) [inline about.html:131]: - [ ] Kindly note that the logo should be aligned to the left and text aligned to the right😎

File: about.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        <li><a href="#">English</a></li>
+      </ul>
+    </nav>
+    <button class="menu-toggle" type="button">
+      <img src="images/menu-hambuger1.svg" alt="menu hamburger" aria-hidden="true">
+    </button>
+    <nav class="second-navbar">
+      <a href="./index.html" class="muson-logo"><img src="images/home-logo.jpg" alt="muson logo"></a>
+      <ul class="second-nav-items">
+        <li><a class="nav-links" href="./about.html">About</a></li>
```
Comment (rotimiazeez) [inline about.html]: - [ ] kindly add a link to navigate to the `home page` instead of having an about here🤷

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @chukwuemeka1234 👋, 

## Good job so far!
### To Highlight🔥
- Good commit message👍
- Descriptive PR message👌
- Good README💯
- No Linter errors✔
- Great UI🎉
### What a wonderful project you have here👏, kindly make the suggested changes and you'll be flying to the moon and beyond🚀💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (chukwuemeka1234) [reply about.html:133]: Thank you for the review. The background of my footer has a white background for both mobile and desktop screens. I can't see a dark background.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔**
Hello @chukwuemeka1234 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**